vscode/src/providers/chat_view_provider.ts

```diff
@@ -112,39 +112,39 @@ export class FlutterGPTViewProvider implements vscode.WebviewViewProvider {
                         break;
                     }
                 case "checkKeyIfExists":
-                    {
-                        this._checkIfKeyExists();
-                        break;
-                    }
+                {
+                    this._checkIfKeyExists();
+                    break;
+                }
                 case "dashResponse":
                     {
                         const { agent, data: _data, messageId, buttonType } = JSON.parse(data.value);
                         console.log('agent', buttonType, _data, messageId, agent);
                         if (agent === "diffView") {
                             const updatedMessage = await DiffViewAgent.handleResponse(buttonType, _data, messageId);
-                            // update the message with messageId
                             if (updatedMessage) {
                                 this._publicConversationHistory[messageId] = updatedMessage;
                                 this._view?.webview.postMessage({ type: 'displayMessages', value: this._publicConversationHistory });
                             }
                         }
+                        break;
                     }
 
             }
         });
 
         webviewView.onDidChangeVisibility(() => {
+            console.log('webview', webviewView.visible);
             if (webviewView.visible && this._view) {
                 this._view?.webview.postMessage({ type: 'focusChatInput' });
+                this._view?.webview.postMessage({ type: 'shortCutHints', value: shortcutInlineCodeRefactor() });
```

Comment:
@wadhia-yash The shortcut keys are empty in display. 
<img width="653" alt="Screenshot 2024-03-11 at 8 07 28 PM" src="https://github.com/Welltested-AI/dash-ai/assets/56000318/fa94120b-4ace-4aaa-b0f1-8671e818c26b">


Also does this help to know that the tippy for code attachment always works. Only the tippy for hint icons in the bottom fails. what could be a possible explanation.